update GCE instance image with Debian 8 Jessie

dminca opened the pull request:
* debian-7 is marked as "DEPRECATED" in Google Compute Engine Images
  * as a result, by default use `debian-8` Jessie

Resolves:
Related:
Signed-off-by: Daniel Andrei Minca <mandrei17@gmail.com>

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - ~Feature Pull Request~
 - ~New Module Pull Request~
 - :white_check_mark: Bugfix Pull Request
 - :white_check_mark: Docs Pull Request

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->
**cloud / google / gce** module

##### ANSIBLE VERSION
<!--- Paste verbatim output from “ansible --version” between quotes below -->
```
ansible 2.2.1.0
  config file = /home/dminca/.ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->
* `debian-7` was marked as **Deprecated** in Google Compute Engine
  * as a result, use `debian-8` Jessie, as it's latest stable image

![selection_006](https://cloud.githubusercontent.com/assets/5355219/22201509/9a7fabf4-e16c-11e6-9072-ac52049185d9.png)

<!---
If you are fixing an existing issue, please include "Fixes #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->

<!-- Paste verbatim command output below, e.g. before and after your change -->
```

```

Commit:
update instance image with Debian 8 Jessie

* debian-7 is marked as "DEPRECATED" in Google Compute Engine Images
  * as a result, by default use `debian-8` Jessie

Resolves:
Related:
Signed-off-by: Daniel Andrei Minca <danielandrei.minca@gameloft.com>

gundalow commented:
I wonder if this will catch users of this module out? If their installs changed from Debian 7 to Debian 8 when updating from Ansble 2.2 to 2.3.

Would it be worth having the description for this option say it will generally be updated to follow the latest stable release?

Commit:
update docs regarding latest stable default image

* after speaking with @gundalow, we decided it's better for the users to
  know that the default image will follow latest stable debian by GCE

Resolves: #20558
Related:
Signed-off-by: Daniel Andrei Minca <danielandrei.minca@gameloft.com>

gundalow reviewed (approved):
shipit
